### PR TITLE
refactor and fix base64decode panic

### PR DESCRIPTION
**Pull request by xiekeyi98**
Fix https://github.com/pingcap/tidb-ctl/issues/15

### CODE REVIEW

**Comment by winkyao** [comment]
@jackysp PTAL

**Comment by jackysp** [comment]
PTAL @imtbkcat

File: cmd/base64decode.go [hunk 1]
```diff
@@ -28,26 +26,26 @@ var (
...
+	uDec, err := base64Decode(inputValue)
 	if err != nil {
 		return err
 	}
 	if len(uDec) <= 8 {
 		var num uint64
-		hexStr := hex.EncodeToString(uDec)
-		fmt.Printf("hex: %s\n", hexStr)
-		err = binary.Read(bytes.NewBuffer(uDec[0:8]), binary.BigEndian, &num)
+		fmt.Printf("hex: %s\n", uDec)
```
Comment (imtbkcat) [inline cmd/base64decode.go]: can not print hex value here

```
./tidb-ctl base64decode AAAAACqPhb0=
hex: *���
uint64: 714048957
```

File: cmd/base64decode.go [hunk 2]
```diff
@@ -57,5 +55,5 @@ func base64decode(_ *cobra.Command, args []string) error {
 }
 
 func init() {
-	base64decodeCmd.Flags().StringVarP(&inputValue, "value", "v", "", "the value you want decode")
+	newBase64decodeCmd.Flags().StringVarP(&inputValue, "value", "v", "", "the value you want decode")
```
Comment (imtbkcat) [inline cmd/base64decode.go]: flag should be changed for updating document.

File: cmd/global_test.go [hunk 3]
```diff
@@ -10,18 +10,18 @@ func Test(t *testing.T) {
...
-func (s *EtcdCtl) TestBase64Encode(c *C) {
+func (s *Global) TestBase64Encode(c *C) {
 	c.Parallel()
 	expected := "SGVsbG8sV29ybGQ="
 	obtained := base64Encode("Hello,World")
 	c.Assert(obtained, Equals, expected)
 }
 
-func (s *EtcdCtl) TestBase64Decode(c *C) {
+func (s *Global) TestBase64Decode(c *C) {
```
Comment (imtbkcat) [inline cmd/global_test.go:18]:
```
./tidb-ctl base64decode -v cAAAAAABX+s=
hex: 7000000000015feb
uint64: 8070450532248018923
```

adding test for this value is better.

File: cmd/base64decode.go (hunk 1 above)
Comment (xiekeyi98) [reply cmd/base64decode.go]: DONE.
```
./tidb-ctl base64decode AAAAACqPhb0=
hex: 000000002a8f85bd
uint64: 714048957
```

File: cmd/global_test.go (hunk 3 above)
Comment (xiekeyi98) [reply cmd/global_test.go:18]: I added test in https://github.com/pingcap/tidb-ctl/blob/1dec121f921d1abae4dcaf0c4a9d2070ddaa895f/cmd/cmdctl_test.go#L41

File: cmd/base64decode.go (hunk 2 above)
Comment (xiekeyi98) [reply cmd/base64decode.go]: I have deleted this flag.
Because I think this flag is a little redundant.

**Review by imtbkcat — DISMISSED** [review]
LGTM

**Comment by xiekeyi98** [comment]
PTAL @winkyao @jackysp

**Review by jackysp — APPROVED** [review]
LGTM

**Pull request merged**